test/misc.jl

```diff
@@ -23,3 +23,35 @@ using SpecialFunctions, StatsFuns
         end
     end
 end
+
+@testset "logmvbeta" begin
+    @testset "symmetry" for eltya in (Float32, Float64)
+                            for eltyb in (Float32, Float64)
+            #  Bᵢ(a, b) = Bᵢ(b, a)
+            for p in 1:50
+                a = rand(eltya)
+                b = rand(eltyb)
+                @test logmvbeta(p, a, b) ≈ logmvbeta(p, b, a)
+            end
+        end
+    end
+
+    @testset "consistent with logbeta" for eltya in (Float32, Float64)
+                                           for eltyb in (Float32, Float64)
```

Comment:
Wrong indentation. Same below.

Reply:
Which?

(Sorry for being daft. I haven't been Julia-ing for a while.)

Reply:
I've fixed it directly.